docs/conf.py

```diff
@@ -65,20 +68,38 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
+if tags.has('release'):
+    exclude_patterns.append('developer/*')
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+if ON_RTD:
+    html_theme = 'sphinx_rtd_theme'
+else:
+    html_theme = 'sphinx_scality'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
 html_theme_options = {
     'logo_only': True,
+    'social_links': [
+        ("linkedin", "https://www.linkedin.com/company/scality/"),
+        ("twitter", "https://twitter.com/scality"),
+        ("instagram", "https://instagram.com/scalitylife"),
+        ("facebook", "https://www.facebook.com/scality/"),
+    ],
+    'footer_links': [
+        ("Support", "https://support.scality.com"),
```

Comment:
It'd be nice for this link to be to Github for open-source builds.

Reply:
Done